### PR TITLE
CEDS-2327 Added defaultInputClasses parameter

### PR DESCRIPTION
**Pull request by alsahid1993**
defaultInputClasses parameter was added to override exportsInputText's inputClasses

### CODE REVIEW

File: app/views/declaration/exporter_details.scala.html [hunk 1]
```diff
@@ -68,51 +68,57 @@
...
-            labelKey = "declaration.exporter-detail.eori"
-        )
+    @or
 
-        @or
+    @exportsName
 
-        @exportsName
 
         @govukFieldset(Fieldset(
```
Comment (SteveSugden) [inline app/views/declaration/exporter_details.scala.html:112]: Could you fix the indentation here?

File: conf/messages.en [hunk 2]
```diff
@@ -958,6 +958,7 @@ declaration.commodityMeasure.units.hint = Enter weight in kilograms
 
 declaration.summary.locations.header = Locations
 declaration.summary.parties.header = Parties
+supplementary.summary.parties.header = Parties
```
Comment (SteveSugden) [inline conf/messages.en:961]: What's this new key for? I don't think you are using it.  In general we are trying to get rid of the "supplementary...." message keys.

File: build.sbt [hunk 3]
```diff
@@ -31,13 +31,7 @@ lazy val microservice = Project(appName, file("."))
...
     parallelExecution in IntegrationTest := false
   )
-  .settings(
-    resolvers ++= Seq(
-      Resolver.bintrayRepo("hmrc", "releases"),
-      Resolver.jcenterRepo,
-      Resolver.bintrayRepo("emueller", "maven")
-    )
-  )
+  .settings(resolvers ++= Seq(Resolver.bintrayRepo("hmrc", "releases"), Resolver.jcenterRepo, Resolver.bintrayRepo("emueller", "maven")))
```
Comment (SteveSugden) [inline build.sbt:34]: Did you mean to change this file?  (I think the contents are the same but the fact that it's in this PR makes me twitchy).

File: app/views/components/gds/exportsInputText.scala.html [hunk 4]
```diff
@@ -27,9 +27,9 @@
         isPageHeading: Boolean = false,
         headingClasses: String = gdsPageLabel,
         labelClasses: String = "govuk-label--m",
+        defaultInputClasses: String = "govuk-!-width-two-thirds",
```
Comment (SteveSugden) [inline app/views/components/gds/exportsInputText.scala.html:30]: This solution looks OK to me.  What do other devs think?

File: build.sbt (hunk 3 above)
Comment (alsahid1993) [reply build.sbt:34]: I merged the master to my branch & resolved the merge conflicts, which was how this was added in

File: conf/messages.en (hunk 2 above)
Comment (alsahid1993) [reply conf/messages.en:961]: key has been removed

File: app/views/components/gds/exportsInputText.scala.html (hunk 4 above)
Comment (alsahid1993) [reply app/views/components/gds/exportsInputText.scala.html:30]: @PatrykRudnicki @PauloPortugal

File: app/views/declaration/exporter_details.scala.html [hunk 5]
```diff
@@ -68,18 +68,19 @@
...
+        @messages("declaration.exporter-detail.addressHeading")
     </legend>
 }
 
 @or = {
     <p class="govuk-body">Or</p>
 }
 
+
+
```
Comment (PatrykRudnicki) [inline app/views/declaration/exporter_details.scala.html:80]: Please remove unnecessary lines

File: app/views/declaration/exporter_details.scala.html [hunk 6]
```diff
@@ -95,14 +96,15 @@
...
-            labelKey = "declaration.exporter-detail.eori"
+        field = form("details.eori"),
+        labelKey = "declaration.exporter-detail.eori"
         )
 
         @or
 
         @exportsName
 
+
```
Comment (PatrykRudnicki) [inline app/views/declaration/exporter_details.scala.html:107]: The same like Steve. Could you remove this line?

File: app/views/declaration/exporter_details.scala.html [hunk 7]
```diff
@@ -95,14 +96,15 @@
         @pageHint
 
         @exportsInputText(
-            field = form("details.eori"),
-            labelKey = "declaration.exporter-detail.eori"
+        field = form("details.eori"),
+        labelKey = "declaration.exporter-detail.eori"
```
Comment (PatrykRudnicki) [inline app/views/declaration/exporter_details.scala.html:100]: Another style comment. Could you use the previous version?

File: build.sbt (hunk 3 above)
Comment (PatrykRudnicki) [reply build.sbt:34]: This is just style change. It looks exactly the same like before but in one line

**Pull request merged**